protocol.html

```diff
@@ -384,19 +384,29 @@ <h3 property="schema:name">Overview</h3>
                 <div datatype="rdf:HTML" property="schema:description">
                   <p><em>This section is non-normative.</em></p>
 
-                  <p>The following diagrams show the high-level interactions involved in <em>source</em> and <em>target</em> flows.
+                  <p>The following diagram shows the high-level interactions involved discovering and establishing a notification subscription.
                      How a client retrieves an access token for step 3 is outside the scope of this document.</p>
 
+                  <figure id="solid-notifications-flow-protocol" rel="schema:hasPart" resource="#solid-notifications-flow-protocol">
+                    <img src="notifications-flow-protocol.mmd.svg" rel="schema:image" width="800" />
+
+                    <figcaption property="schema:name">Solid Notifications Flow</figcaption>
+                  </figure>
+
+                  <p>Subscription types in which a client establishes a subscription and then maintains a persistent connection to a Notifications Source are illustrated by the following diagram.</p>
+
                   <figure id="solid-notifications-flow-source" rel="schema:hasPart" resource="#solid-notifications-flow-source">
                     <img src="notifications-flow-source.mmd.svg" rel="schema:image" width="800" />
 
-                    <figcaption property="schema:name">Solid Notifications Flow (source)</figcaption>
+                    <figcaption property="schema:name">Solid Subscription Flow for Connected Clients</figcaption>
                   </figure>
 
+                  <p>Subscription types in which a client establishes a subscription and does not maintain a persistent connection to a Notifications Source are illustrated by the following diagram.</p>
+
                   <figure id="solid-notifications-flow-target" rel="schema:hasPart" resource="#solid-notifications-flow-target">
                     <img src="notifications-flow-target.mmd.svg" rel="schema:image" width="800" />
 
-                    <figcaption property="schema:name">Solid Notifications Flow (target)</figcaption>
+                    <figcaption property="schema:name">Solid Notifications Flow for Offline Clients</figcaption>
```

Comment:
I don't think *Offline Clients* makes sense here.
Actually, client-server terminology might be confusing since the Subscriber acts as the server when a notification gets delivered, and the notifications publisher will act as a client.

Actually, the Subscriber needs to maintain their `target` online to be able to receive incoming requests with notifications.

Reply:
```suggestion
                    <figcaption property="schema:name">Solid Notifications Flow, where the publisher connects to the notification target (from the <a href="#subscription-request">subscription request</a>)</figcaption>                    
```

again `target` chan be changed in dedicated PR